ios/FluentUI/Controls/SegmentedControl.swift

```diff
@@ -354,6 +442,14 @@ open class SegmentedControl: UIControl {
         invalidateIntrinsicContentSize()
     }
 
+    open func getViewForButton(at index: Int) -> UIView? {
+        guard index <= buttons.count && style != .tabs else {
```

Comment:
no need to only make it happen for tab style i think. also, should we make this objc friendly?

Reply:
Is there a clear requirement for this scenario? Making it open and @objc visible will increase the static lib size.